washington_a03b.js

```diff
@@ -0,0 +1,61 @@
+/*
+Timothy Washington
+washington_a03b.js
+INFO 2124 - Javascript I
+Thoendel
+Creation date: Dec 22, 2019
+*/
+//comment Legend: {c=creates variable, s=sets a variable to a value}
+//Note: Copy Paste doesn't work through virtual machine. Unless you're expecting us to open the assignment on this itself on this browser (Which, mind you, is a laggy mess), we find workarounds that take more steps or simply type the whole thing out anyways.
```

Comment:
I know copy and paste don't work. I've had good luck with working with a browser on the vm - or i code on a local machine and transfer files with git. Sorting out a workflow that works for you is a good practice. VM development is a pretty common thing - its good to know how to work with that.